Libraries/Image/Image.android.js

```diff
@@ -131,6 +132,12 @@ var Image = React.createClass({
      */
     testID: PropTypes.string,
     /**
+     * The mechanism that should be used to resize the image when the image's dimensions
+     * differ from the image view's dimensions. 'auto' means that some heuristics will
+     * be used to pick between 'resize' and 'scale'. The default value is 'auto'.
+     */
```

Comment:
I might be mistaken but I think we only look at *.ios.js when generating docs for the website: http://facebook.github.io/react-native/docs/image.html

You can verify locally using:

    cd react-native/website
    npm install
    npm start
    (prints a localhost URL)
    (later rm -rm website/node_modules - take up space and I think when you run Flow it will analyze the folder which is not really needed)

By adding `@platform android` as the last line of the comment the prop will be displayed with an Android flag in the docs.

nit: Could you please explain what the difference between `resize` and `scale` is? I don't actually remember which one does the downsampling by skipping pixels.